### PR TITLE
Making the landing page more friendlier 

### LINKED ISSUE #6615: Make the landing page more friendly
When you land on docs.sunpy.org, you're currently greeted with two boxes with bullet points. We should instead adapt what is currently in the brief tour into some sort of introduction into what sunpy can do. The user should be greeted with what sunpy can do, ideally in graphical format. This could be either in the form of actual plots or something more cartoon-y as in the glyph attached below.

sunpy-summary-slide

### PR DESCRIPTION
**Pull request by Satan-Claws**
I've added the svg graphic provided by @wtbarnes and have added a one-liner that describes what SunPy does to new beginners. I can add further messages if needed.

closes #6615

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -3,8 +3,11 @@ sunpy core Documentation
 ************************
 
 ``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.
+It can be used for downloading solar data, such as maps and time series, it can also be used to calculate coordinates among other uses.
 
-
+.. image:: tour.svg
+  :width: 700
+  :align: center
```
Comment (nabobalis) [inline docs/index.rst:10]:
```suggestion
  :align: center

```

File: docs/index.rst [hunk 2]
```diff
@@ -3,8 +3,11 @@ sunpy core Documentation
 ************************
 
 ``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.
+It can be used for downloading solar data, such as maps and time series, it can also be used to calculate coordinates among other uses.
```
Comment (nabobalis) [inline docs/index.rst:6]:
```suggestion
It provides search and download functionality, data containers for image and time series data, as well as commonly used coordinate frames and transformations between such frames.
```

File: docs/index.rst [hunk 3]
```diff
@@ -3,7 +3,11 @@ sunpy core Documentation
 ************************
 
 ``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.
+It provides search and download functionality, data containers for image and time series data, as well as commonly used coordinate frames and transformations between such frames.
 
+.. image:: tour.svg
+  :width: 700
```
Comment (nabobalis) [inline docs/index.rst:9]: Do we need to add this? I just wonder how this will work on smaller displays.

Comment (Satan-Claws) [reply docs/index.rst:9]: I think it'd be fine with 300px too, I just checked on my local setup

Comment (nabobalis) [reply docs/index.rst:9]: What if we just remove this?

Comment (Satan-Claws) [reply docs/index.rst]: Yeah I think that'll work too

**Comment by Satan-Claws** [comment]
@nabobalis what should I do about the pre-commit.ci check that's failing?

**Comment by nabobalis** [comment]
You will want to fix it, see https://docs.sunpy.org/en/latest/dev_guide/contents/code_standards.html#formatting for more infomation.

**Comment by Satan-Claws** [comment]
@nabobalis Pre-Commit now passes. I'm sorry I didn't know about Coverage and Pre-Commit issues before so I couldn't work around that.

**Comment by nabobalis** [comment]
Why was the image replaced with a png?

**Comment by Satan-Claws** [comment]
> Why was the image replaced with a png?

I checked online and it seems pre-commit has issues with svg formatting.

**Comment by nabobalis** [comment]
> > Why was the image replaced with a png?
> 
> I checked online and it seems pre-commit has issues with svg formatting.

That is odd, if it is, we should make the pre-commit ignore the file instead of replacing it with a png.

**Comment by Satan-Claws** [comment]
> That is odd, if it is, we should make the pre-commit ignore the file instead of replacing it with a png.

Can you guide me how to do that? I'll upload the svg back and delete the png.

**Comment by nabobalis** [comment]
> @nabobalis Pre-Commit now passes. I'm sorry I didn't know about Coverage and Pre-Commit issues before so I couldn't work around that.

Coverage doesn't matter for this style of change.

**Comment by nabobalis** [comment]
> > That is odd, if it is, we should make the pre-commit ignore the file instead of replacing it with a png.
> 
> Can you guide me how to do that? I'll upload the svg back and delete the png.

What happens if you run the pre-commit on the SVG?

**Comment by Satan-Claws** [comment]
> What happens if you run the pre-commit on the SVG?

I looked into details at the pre-commit fail and it showed error in the .svg file. I checked online and they said you should globally exclude .svg files.

**Comment by nabobalis** [comment]
> > What happens if you run the pre-commit on the SVG?
> 
> I looked into details at the pre-commit fail and it showed error in the .svg file. I checked online and they said you should globally exclude .svg files from pre-commit checks.

But what happens if you run the precommit, does it not fix the error?

**Comment by Satan-Claws** [comment]
> But what happens if you run the precommit, does it not fix the error?

It showed this error 

diff --git a/docs/tour.svg b/docs/tour.svg
index 4d4a212..d0b6c76 100644
--- a/docs/tour.svg
+++ b/docs/tour.svg
@@ -1 +1 @@
-<svg version="1.1" viewBox="0.0 0.0 960.0 540.0" fill="none" stroke="none" stroke-linecap="square" stroke-miterlimit="10" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg"><clipPath id="gde43c87251_0_39.0"><path d="m0 0l960.0 0l0 540.0l-960.0 0l0 -540.0z" clip....

(output truncated)

If it's possible to make it such that the .svg file is ignored by pre-commit I'll do that, just let me know how

**Comment by nabobalis** [comment]
> > But what happens if you run the precommit, does it not fix the error?
> 
> It showed this error
> 
> diff --git a/docs/tour.svg b/docs/tour.svg index 4d4a212..d0b6c76 100644 --- a/docs/tour.svg +++ b/docs/tour.svg @@ -1 +1 @@ -<path d="m0 0l960.0 0l0 540.0l-960.0 0l0 -540.0z" clip....
> 
> (output truncated)

But is this an error? I can't tell from this alone.

> If it's possible to make it such that the .svg file is ignored by pre-commit I'll do that, just let me know how

We can add it to be ignored but I don't see why we would need to without further investigation.

**Comment by Satan-Claws** [comment]
> We can add it to be ignored but I don't see why we would need to without further investigation.

I was referring to this https://github.com/OCA/oca-addons-repo-template/issues/25

**Comment by nabobalis** [comment]
> > We can add it to be ignored but I don't see why we would need to without further investigation.
> 
> I was referring to this [OCA/oca-addons-repo-template#25](https://github.com/OCA/oca-addons-repo-template/issues/25)

Fair enough, I don't really understand but I think we should add back the SVG file and work out the rest from there.

**Comment by Satan-Claws** [comment]
> Fair enough, I don't really understand but I think we should add back the SVG file and work out the rest from there.

Ok, I'm adding the .svg back, removing the .png

**Comment by nabobalis** [comment]
pre-commit.ci autofix

**Comment by nabobalis** [comment]
Now that is not an error I expected to see.

**Comment by nabobalis** [comment]
Here, https://github.com/sunpy/sunpy/blob/main/setup.cfg#L276 can you add a skip for all svg files please?

**Comment by Satan-Claws** [comment]
> Here, https://github.com/sunpy/sunpy/blob/main/setup.cfg#L276 can you add a skip for all svg files please?

https://github.com/sunpy/sunpy/pull/6792

**Comment by Satan-Claws** [comment]
> #6792
Wait, I should've added it this to PR, I'll close and edit again.

**Comment by nabobalis** [comment]
That has done it!

**Comment by Satan-Claws** [comment]
@nabobalis will we ignore the Coverage issue? And how should I go on from now on to avoid it?

**Comment by nabobalis** [comment]
> @nabobalis will we ignore the Coverage issue? And how should I go on from now on to avoid it?

Yes, we will ignore it. This change does not affect our coverage. There are no changes to the Python code or tests. 

We have also not run the entire test suite so far which breaks the coverage reporting.

**Comment by Satan-Claws** [comment]
Alright, Thanks for helping me understanding how things operate around here. Hope to be of help in the future. :D

**Comment by Satan-Claws** [comment]
@nabobalis should this be merged? Or is there anything else I should add?

**Comment by nabobalis** [comment]
> @nabobalis should this be merged? Or is there anything else I should add?

This will be merged after two people review it.

File: docs/index.rst [hunk 4]
```diff
@@ -3,7 +3,10 @@ sunpy core Documentation
 ************************
 
 ``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.
+It provides search and download functionality, data containers for image and time series data, as well as commonly used coordinate frames and transformations between such frames.
 
+.. image:: tour.png
```
Comment (ayshih) [inline docs/index.rst:8]: Switch back to `tour.svg`

**Review by ayshih — CHANGES_REQUESTED** [review]

**Comment by Satan-Claws** [comment]
@ayshih ohh sorry sorry, I'll do it as soon as I get on pc

**Comment by Satan-Claws** [comment]
@ayshih I've done it. I'll look into the other PR in some time I'm not with my PC right now.

**Review by ayshih — APPROVED** [review]
Looks good to me

**Review by nabobalis — APPROVED** [review]

**Comment by nabobalis** [comment]
Thanks for the PR @Satan-Claws

**Pull request merged**